pennylane_lightning/core/src/gates/Constant.hpp

```diff
@@ -196,6 +285,46 @@ namespace Pennylane::Gates::Constant {
     std::pair<GateOperation, size_t>{GateOperation::DoubleExcitationPlus, 4},
 };
 
```

Comment:
A typedef can probably help here too.
I will not comment on all blocks, you know the drill.